Source/CarthageKit/GitHub.swift

```diff
@@ -266,6 +284,14 @@ internal struct GitHubCredentials {
 	}
 }
 
+internal func loadGitHubCredentials(accessToken: GitHubAccessToken?) -> SignalProducer<GitHubCredentials?, CarthageError> {
+	if let accessToken = accessToken {
+		return SignalProducer(value: AccessTokenGitHubCredentials(accessToken: accessToken))
```

Comment:
Yeah we can definetly do that. My retaionale behind this was that for me the command line app is the one responsabile for obtaining the input (user or env) and feed that input to the CarthageKit lib.